CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ project(AnalyzeObjectLabelMap
 set(AnalyzeObjectLabelMap_LIBRARIES AnalyzeObjectLabelMap)
 
 if(NOT ITK_SOURCE_DIR)
-  find_package(ITK 5.1 REQUIRED)
+  find_package(ITK REQUIRED)

```

Comment:
What is the minimum version required?  This indicates that ITK 1.0 should work, but that is not true.  I know that at least 5.0 is required.  

I don't know how far back this is supported.

Reply:
I am pretty sure 5.0, but I cannot say for certain. I can test previous versions until it breaks.

Reply:
It's not worth the effort.  Just set it to 5.0 and call it good.

Reply:
It been resolved.